packages/scoped-custom-element-registry/test/Element.test.html.js

```diff
@@ -104,4 +109,41 @@ describe('Element', () => {
       expect($el.firstElementChild).to.not.be.instanceof(CustomElementClass);
     });
   });
+
+  describe('attributes', () => {
+    it('should call setAttribute', () => {
+      const {tagName, CustomElementClass} = getObservedAttributesTestElement(['foo']);
+      customElements.define(tagName, CustomElementClass);
+      const $el = getHTML('<div></div>');
+      $el.innerHTML = `<${tagName}></${tagName}>`;
+
+      $el.firstElementChild.setAttribute('foo', 'bar');
+
+      expect($el.firstElementChild.getAttribute('foo')).to.equal('bar');
```

Comment:
And then the `.firstElementChild` lookups could be removed down below here and in the other tests:

```suggestion
      $el.setAttribute('foo', 'bar');

      expect($el.getAttribute('foo')).to.equal('bar');
```